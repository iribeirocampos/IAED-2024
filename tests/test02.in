p parque1 10 0.30 0.50 15.00
p "Parque 2" 20 0.30 0.50 15.00
p Parque3 10 0.25 0.40 20.00
p
q
