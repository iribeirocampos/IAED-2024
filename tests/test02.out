parque1 10 10
Parque 2 20 20
Parque3 10 10
